app/commands/v2/put_content.rb

```diff
@@ -221,6 +222,14 @@ def update_content_item(content_item)
         content_item.save!
       end
 
+      def update_last_edited_at_if_needed(content_item, last_edited_at = nil)
+        if last_edited_at.nil? && %w(major minor).include?(payload[:update_type])
+          last_edited_at = Time.zone.now
```

Comment:
I am happy to make the `last_edited_at.nil?` check explicit and make it read like the desired behaviour: 
`if last_edit_at_specified? && major_or_minor_update?` etc